pallets/order-book/src/benchmarking.rs

```diff
@@ -351,10 +355,30 @@ benchmarks! {
     }
 
     change_orderbook_status {
+        let order_book_id = OrderBookId::<AssetIdOf<T>> {
+            base: VAL.into(),
+            quote: XOR.into(),
+        };
+
+        create_and_fill_order_book::<T>(order_book_id);
```

Comment:
Maybe use `create_empty_order_book` instead (this benchmark doesn't seem to be affected by placed orders)?

Also I noticed that there are two `create_and_fill_order_book` functions that don't seem to be different (correct me if I'm wrong); maybe we should remove one of them?

Reply:
1. It looks better when we benchmark it with filled order book.
2. We have it in `benchmarking.rs` & `test_utils.rs`. Benchmarking versrion depends on T, the test version depends directly on Runtime. The idea is that benchmarks could be run as benchmarks and as tests. In order book we could use it only as benchmarks and cannot as tests for a while. If we want to use only 1 function we should move it out o the test scope. It wil look like test util outside test module. I think we could decide what to do with this function later when we have a new solution for benchmarks/tests/mocks or write some playground pallet for QA. After that it could be more clear how we can rework it.